jobs/create-relval-jobs.py

```diff
@@ -37,6 +37,7 @@ def createJob(workflow, cmssw_ver, arch):
     e, o = run_cmd(cmd)
     if e:
         print("ERROR:%s:%s" % (workflow, o))
+        exit(1)
```

Comment:
@iarspider , I do not think we want to add this. The command above can fail e.g. it can fail to run das-client for some workflows but we still want to run other workflows. So please remove this

Reply:
is `exit` here going to terminate the main python process or just the thread?

Reply:
looking at the code in the `try` block, I am sure we do not want to exit. So please remove it

Reply:
jobscheduler should never fail because of the commands it runs. commands/relvals failures are reported in the IB dashboard via the workflow.log file we create in the `try` block

Reply:
by the way, does runTheMatrix create `runall-report-step123-.log` for the workflow if it is asked to use GPU and n GPU is found?

Reply:
ok, we can not just simple exit from here on failure. This code runs the relval step1 (to run the das-client) and create cmdLog. We do not want to exit the relval jenkins jobs as one das-cleint step failed.

I suggest to remove the exit so that we can move forward with this PR. 
Later we can add checks in run-ib-relval.py to process the jobs/commands and fail if needed

Reply:
Sure. For the record: in this state, the Jenkins job will not fail if GPU is required but not found, but will also not run any workflows.

Reply:
sure but I think even with `exit(1)` will not fail the job as this only exists the thread `create-relval-jobs.py` will still exit with exit code `0` :-)